script/Ciel/Day 14/14_01B_03_CIEL14B_4_1B.txt

```diff
@@ -113,7 +113,7 @@ It feels good, yet terrifying.
 [sha:a407fde5b4a8d13ad076cb64211e4da7dacbc2af]{
 -- Page 25, Offset 40320.
 -- 相反する指向性、背中合わせのロジックエラーが、
-These conflicting directives,
+These conflicting directives, 
```

Comment:
Space needs to go at the left hand side of the following line; trailing whitespace is stripped by the tooling